mypy/version.py

```diff
@@ -5,12 +5,21 @@
 # - Release versions have the form "0.NNN".
 # - Dev versions have the form "0.NNN+dev" (PLUS sign to conform to PEP 440).
 # - For 1.0 we'll switch back to 1.2.3 form.
-__version__ = '0.940+dev'
-base_version = __version__
+base_version = '0.940+dev'
+# Overridden by setup.py
+__version__ = base_version
 
-mypy_dir = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
-if __version__.endswith('+dev') and git.is_git_repo(mypy_dir) and git.have_git():
-    __version__ += '.' + git.git_revision(mypy_dir).decode('utf-8')
-    if git.is_dirty(mypy_dir):
-        __version__ += '.dirty'
-del mypy_dir
+def setup_compute_version() -> str:
+    # We allow an environment variable to override version, but we should probably
+    # enforce that it is consistent with the existing version minus additional information.
+    if "MYPY_VERSION" in os.environ:
+        assert os.environ["MYPY_VERSION"].hasprefix(base_version)
```

Comment:
```suggestion
        assert os.environ["MYPY_VERSION"].startswith(base_version)
```

Reply:
uhh yeah oops. Sorry I couldn't figure out how to test this without CI and how to create CI without a noisy PR